Sourcery/Models/Type.swift

```diff
@@ -169,7 +169,6 @@ public class Type: NSObject, SourceryModel, Annotated {
     }
 
     // sourcery: skipEquality, skipDescription
-    /// Contained types groupd by their names
     public private(set) var containedType: [String: Type] = [:]
```

Comment:
I prefer the strategy to fail earlier just because it will push contributor better to keep the docs updated 😄